### PR TITLE
feat: Add SimCard step for 9151

### PR DESCRIPTION
**Pull request by boundlesscalm**

### CODE REVIEW

File: src/features/flows/nRF9151/SimCard.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            <p>There are two pre-activated SIM cards in the kit:</p>
+            <br />
+            <ol className="tw-list-outside tw-list-disc tw-pl-4">
+                <li>
+                    <Link
+                        label="SIM card from Onomondo"
+                        href="https://onomondo.com/go/nordic-dev-kit/#form"
+                        color="tw-text-primary"
+                    />{' '}
+                    with 10 MB free data.
```
Comment (martiles) [inline src/features/flows/nRF9151/SimCard.tsx:26]: activated with 10 MB of free data.

File: src/features/flows/nRF9151/SimCard.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    insight tool.
+                </li>
+                <br />
+                <li>
+                    <Link
+                        label="SIM card from Wireless Logic"
+                        href="https://www.wirelesslogic.com/simclaim/nsctrial/"
+                        color="tw-text-primary"
+                    />{' '}
+                    with 5 MB free data for 6 months.
```
Comment (martiles) [inline src/features/flows/nRF9151/SimCard.tsx:39]: activated with 5 MB of free data.

File: src/features/flows/nRF9151/SimCard.tsx (hunk 1 above)
Comment (boundlesscalm) [reply src/features/flows/nRF9151/SimCard.tsx:26]: It does talk about the sim cards being "pre-activated" above the 2 sim card entries.
Not sure if we should duplicate this mention?

Comment (martiles) [reply src/features/flows/nRF9151/SimCard.tsx:26]: true, avoid this

File: src/features/flows/nRF9151/SimCard.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            <ol className="tw-list-outside tw-list-disc tw-pl-4">
+                <li>
+                    <Link
+                        label="SIM card from Onomondo"
+                        href="https://onomondo.com/go/nordic-dev-kit/#form"
+                        color="tw-text-primary"
+                    />{' '}
+                    with 10 MB free data.
+                    <br />
+                    If you register the SIM card, you will get additional 40 MB
```
Comment (martiles) [inline src/features/flows/nRF9151/SimCard.tsx:28]: If you register the SIM card, you will receive an additional 40 MB to use, and you will have a two-month trial period to test Onomondo's real-time network insight tool.

**Comment by ketile** [comment]
Looks good to me @boundlesscalm

**Pull request merged**